packages/performance/src/services/perf_logger.ts

```diff
@@ -83,18 +82,18 @@ interface TraceMetric {
   counters?: Array<{ key: string; value: number }>;
   custom_attributes?: Array<{ key: string; value: string }>;
 }
+
 /* eslint-enble camelcase */
 
-let logger: Logger | undefined;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+let logger: (resource: {}, resourceType: ResourceType) => void | undefined;
 // This method is not called before initialization.
-function getLogger(): Logger {
-  if (logger) {
-    return logger;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
unnecessary eslint suppression. Is it possible to change the type fo `resource` to `unknown`?

Reply:
Changed.